internal/chart/chart.go

```diff
@@ -78,39 +77,39 @@ func parseManifest(manifest string) ([]unstructured.Unstructured, error) {
 	return results, nil
 }
 
-func getCachedAndCurrentManifest(config *Config, renderChartFunc func(config *Config) (*release.Release, error)) (string, string, error) {
+func getCachedAndCurrentManifest(config *Config, customFlags map[string]interface{}, renderChartFunc func(config *Config, customFlags map[string]interface{}) (*release.Release, error)) (string, string, error) {
 	cachedSpecManifest, err := config.Cache.Get(config.Ctx, config.CacheKey)
 	if err != nil {
 		return "", "", fmt.Errorf("could not get manifest from cache : %s", err.Error())
 	}
 
-	if !shouldRenderAgain(cachedSpecManifest, config) {
+	if !shouldRenderAgain(cachedSpecManifest, config, customFlags) {
 		return cachedSpecManifest.Manifest, cachedSpecManifest.Manifest, nil
 	}
 
-	currentRelease, err := renderChartFunc(config)
+	currentRelease, err := renderChartFunc(config, customFlags)
 	if err != nil {
 		return cachedSpecManifest.Manifest, "", fmt.Errorf("could not render manifest : %s", err.Error())
 	}
 
 	return cachedSpecManifest.Manifest, currentRelease.Manifest, nil
 }
 
-func shouldRenderAgain(spec ServerlessSpecManifest, config *Config) bool {
-	// spec is up-to-date only if flags used to render and manager is the same one who rendered it before
-	equalFlags := reflect.DeepEqual(spec.CustomFlags, config.Release.Flags)
-	return !(spec.ManagerUID == config.ManagerUID && equalFlags)
+func shouldRenderAgain(cachedSpec ServerlessSpecManifest, config *Config, customFlags map[string]interface{}) bool {
+	// cachedSpec is up-to-date only if flags used to render and manager is the same one who rendered it before
+	equalFlags := reflect.DeepEqual(cachedSpec.CustomFlags, customFlags)
+	return !(cachedSpec.ManagerUID == config.ManagerUID && equalFlags)
 }
 
-func renderChart(config *Config) (*release.Release, error) {
+func renderChart(config *Config, customFlags map[string]interface{}) (*release.Release, error) {
```

Comment:
customFlags seems to be a more precise name for this field. 
* We name it `flags` in other places so we should name it like this here.
* Right now we use secrets (as cache) to store info about the last installation and in this secret, we store flags under the field `customFlags`. I think if we decide to change this field to `overrides` then we should update the field in secrets too. But I'm not sure if it would be an easy change because then we have to think about migration from old to new solution. 
* I don't feel different between these names